conda_smithy/ci_register.py

```diff
@@ -174,55 +174,79 @@ def appveyor_configure(user, project):
         raise ValueError(response)
 
 
-def add_project_to_travis(user, project):
+def travis_wait_until_synced(user, ignore=False):
     headers = travis_headers()
     endpoint = 'https://api.travis-ci.org'
+    is_sync_url = '{}/users'.format(endpoint)
+    for c in range(20):
+        response = requests.get(is_sync_url, headers=headers)
+        content = response.json()
+        print(".", end="", flush=True)
```

Comment:
Appears there is no `flush` for Python 2.

ref: http://stackoverflow.com/questions/230751/how-to-flush-output-of-python-print/35467658#35467658